libs/contrib/Text/PrettyPrint/Prettyprinter/Doc.idr

```diff
@@ -2,7 +2,7 @@ module Text.PrettyPrint.Prettyprinter.Doc
 
 import Data.List
 import Data.Maybe
-import Data.Strings
+import public Data.Strings
```

Comment:
Looks good, I'm just curious why this has changed to public? Sorry I didn't catch it earlier

Reply:
Good catch, in a previous version the compiler complained without the public re-export. Currently works without, so i'll revert to non-export.